resources/js/Components/DashboardMetrics.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+	<div class="container max-w-6xl px-5 mx-auto" >
+		<div class="grid gap-7 sm:grid-cols-2 lg:grid-cols-4">
+			<div class="p-5 bg-white rounded shadow-sm" v-for="(value, name) in metrics" :key="name">
+				<div class="text-base text-gray-400 capitalize">{{name}}</div>
+				<div class="flex items-center pt-1">
+					<div class="text-2xl font-bold text-gray-900 ">{{value}}</div>
+					<!--<span class="flex items-center px-2 py-0.5 mx-2 text-sm text-green-600 bg-green-100 rounded-full"> 
+                        <svg class="w-4 h-4" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M18 15L12 9L6 15" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path>
+                        </svg>
+					</span>-->
```

Comment:
```suggestion
```
Is there a reason this code is commented out? If its not needed, please remove it.

Reply:
I've removed it, it was unused code from a tailwind component. I wasn't sure if it would be necessary and then forgot to remove it afterwards.